src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,11 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
+        if (UserService.findByEmail(email) != null) {
+            return Objects.equals(UserService.findByEmail(email).getPassword(), password);
+        }
```

Comment:
You can simplify your code by `findByEmail` only once, make a variable with type User and then check password from this variable with provided password. Also avoid static methods.